src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -116,14 +117,16 @@ export function CurrencySearch({
   const native = useNativeCurrency()
 
   const filteredSortedTokensWithETH: Currency[] = useMemo(() => {
-    if (!native) return filteredSortedTokens
+    if (!native || (chainId && [SupportedChainId.CELO, SupportedChainId.CELO_ALFAJORES].includes(chainId))) {
```

Comment:
Why are you excluding celo from including the native currency?

Reply:
It looks like CELO uses an ERC20 from the list for its native currency. Can you add a comment above this line to that effect?